std_msgs/Header header
uint8 status
uint8 WaterLow = 0   # Water is Low
uint8 WaterOK = 1    # Water is OK